### PR TITLE
fix: replace native modules with react-native-uuid

### PR DESCRIPTION
**Pull request by alanjcharles**

### CODE REVIEW

File: packages/core/src/__tests__/uuid.test.ts [hunk 1]
```diff
@@ -1,24 +1,11 @@
...
-
-    const result = getUUID();
+  it('should get a UUID from react-native-uuid', () => {
+    const uuidSpy = jest.spyOn(uuid, 'v4');
 
-    expect(result).toBe(id);
-    (global as any).__DEV__ = oldDevValue;
+    getUUID();
+    expect(uuidSpy).toBeCalled();
   });
```
Comment (oscb) [inline packages/core/src/__tests__/uuid.test.ts:10]: I think we can remove this test. Not sure what it was trying to accomplish besides making sure we were calling a NativeModule method.

**Review by oscb — APPROVED** [review]
Looks good! Props for the fast turnaround on this.
I left a comment about a test we can remove imo, but otherwise this looks perfect

File: packages/core/src/__tests__/uuid.test.ts (hunk 1 above)
Comment (alanjcharles) [reply packages/core/src/__tests__/uuid.test.ts:10]: awesome thank you! I kinda thought so too haha. I'll remove it right now then.

**Comment by alanjcharles** [comment]
#706

**Comment by SchneiderOr** [comment]
Hey why this isn't reflected in master 2.9.1? i still getting this error on dev

**Pull request merged**